node/README.md

```diff
@@ -0,0 +1,28 @@
+# HubSpot-nodejs oauth sample app
+
+This is a sample app for the [hubspot-nodejs SDK](https://www.npmjs.com/package/@hubspot/api-client). Currently, this app focuses on demonstrating the functionality of [OAuth API](https://developers.hubspot.com/docs/api/working-with-oauth) endpoints and their related actions.
+
+- [Using OAuth 2.0 access tokens](https://developers.hubspot.com/docs/api/intro-to-auth)
+- [Get all contacts](https://developers.hubspot.com/docs/api/crm/contacts)
+
+Please see the documentation on [Creating an app in HubSpot](https://developers.hubspot.com/docs/api/creating-an-ap)
```

Comment:
Let's move this paragraph to top level README (excluding node.js related stuff)

Reply:
What about the link to different sdks?

Reply:
We can leave the first sentence as it is and move everything starting from "Currently, this app focuses.."

Reply:
Moved as discussed